components/ProfilePage/UserPageHeader.js

```diff
@@ -217,18 +307,26 @@ exported.fragments = {
       bio
       level
       appId
+
+      email
+      facebookId
+      githubId
+      availableAvatarTypes
```

Comment:
It seems that these fields are only used in `EditAvatarDialog`. In this case I suggest adding these fields to `EditAvatarDialogUserData` fragment in `EditAvatarDialog` instead.